src/main/scala/ai/privado/model/exporter/DataFlow.scala

```diff
@@ -80,6 +80,21 @@ object DataFlowEncoderDecoder {
   implicit val databaseDetailsEncoder: Encoder[DatabaseDetails] =
     deriveEncoder[DatabaseDetails]
 
+  implicit val databaseSchemaDecoder: Decoder[DatabaseSchema] =
+    deriveDecoder[DatabaseSchema]
+  implicit val databaseSchemaEncoder: Encoder[DatabaseSchema] =
+    deriveEncoder[DatabaseSchema]
+
+  implicit val databaseTableDecoder: Decoder[DatabaseTable] =
+    deriveDecoder[DatabaseTable]
+  implicit val databaseTableEncoder: Encoder[DatabaseTable] =
+    deriveEncoder[DatabaseTable]
+
+  implicit val databaseColumnDecoder: Decoder[DatabaseColumn] =
+    deriveDecoder[DatabaseColumn]
+  implicit val databaseColumnEncoder: Encoder[DatabaseColumn] =
+    deriveEncoder[DatabaseColumn]
+
```

Comment:
Move the Database model-related Encoder, Decoder to the same file where the Database models are defined